lib/cdo/activity_constants.rb

```diff
@@ -10,4 +10,7 @@ module ActivityConstants
   MAXIMUM_NONOPTIMAL_RESULT = 29
   FREE_PLAY_RESULT = 30
   BEST_PASS_RESULT = 100
+
+  REJECTED_RESULT = 150
```

Comment:
Good call, fixed in 3a911a98d7d1de8eb852ca015f995e3a395e5f6e and b7213572f9d666acca361d8e31cd3096c64140c8.